doc/api-reference/rest-api.rst

```diff
@@ -127,29 +127,34 @@ Parameters: None
 
 Response:
 
-A JSON object corresponding to a pandas DataFrame. The names of the
-columns are "ID_1", "loss_type", "value", "lossmea", "lossq05",
-"lossq95".
+A JSON object containing:
 
-**********************************
-GET /v1/calc/:calc_id/mmi_tags
-**********************************
+- an 'impact' key containing a pandas DataFrame; the names of the columns are "ID_1", "loss_type",
+  "value", "lossmea", "lossq05", "lossq95".
+- a 'loss_type_description' dictionary containing a description for each loss type.
+
+*****************************************
+GET /v1/calc/:calc_id/get_exposure_by_mmi
+*****************************************
```

Comment:
I prefer `exposure_by_mmi` over `get_exposure_by_mmi`

Reply:
I used `get` for consistency with `get_impact` (to which I added `get` for the reason above. If I find a better name for `impact` I can rename also this.

Reply:
After renaming the `impact` api endpoint, I also removed `get_` here.